tests/xtro-sharpie/Makefile

```diff
@@ -50,7 +50,32 @@ XMAC_PCH = macosx$(OSX_SDK_VERSION)-$(XMAC_ARCH).pch
 $(XMAC_PCH): .stamp-check-sharpie
 	sharpie sdk-db --xcode $(XCODE) -s macosx$(OSX_SDK_VERSION) -a $(XMAC_ARCH) -exclude ClockKit -exclude JavaNativeFoundation
 
-
+XCAT ?= $(TOP)/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/64bits/MacCatalyst/Xamarin.MacCatalyst.dll
+XCAT_GL ?=
+XCAT_ARCH = x86_64
+XCAT_PCH = ios$(MACCATALYST_SDK_VERSION)-macabi-$(XCAT_ARCH).pch
+
+$(XCAT_PCH): .stamp-check-sharpie
+	sharpie sdk-db --xcode $(XCODE) -s ios$(MACCATALYST_SDK_VERSION)-macabi -a $(XCAT_ARCH) \
+	-exclude AGL \
+	-exclude CalendarStore \
+	-exclude Carbon \
+	-exclude ClockKit \
+	-exclude DiscRecordingUI \
+	-exclude GLKit \
+	-exclude ICADevices \
+	-exclude InputMethodKit \
+	-exclude InstallerPlugins \
+	-exclude IOBluetooth \
+	-exclude IOBluetoothUI \
+	-exclude JavaNativeFoundation \
+	-exclude LDAP \
+	-exclude Python \
+	-exclude Quartz \
+	-exclude SecurityInterface \
```

Comment:
are those exclusions to make sharpie work ?
because others do/should exists

Reply:
Yes, this is to make sharpie work (the corresponding headers don't compile with Mac Catalyst).